src/surrogates/inc/InterpolationSurrogateBuilder.h

```diff
@@ -47,7 +47,7 @@ namespace QUESO
     //! Constructor
     /*! We do not take a const& to the data because we want to compute and
         set the values directly. */
-    InterpolationSurrogateBuilder( InterpolationSurrogateData<V,M>& data );
+    InterpolationSurrogateBuilder( InterpolationSurrogateDataSet<V,M>& data );
```

Comment:
Why replace the original one instead of *adding* the functionality?  The difference being I wouldn't have to populate an `std::vector` with one element.  I guess that's not too much of a hassle...

What do you think?